govinfo.install

```diff
@@ -77,12 +77,6 @@ function govinfo_schema() {
         'not null' => TRUE,
         'default' => '',
       ],
-      'last_modified' => [
```

Comment:
What is the reason for the schema change? I do not see a corresponding code change

Reply:
Just removed the duplicate.